t/t3401-module-trace.t

```diff
@@ -0,0 +1,56 @@
+#!/bin/sh
+
+test_description='Test flux module trace'
+
+. $(dirname $0)/sharness.sh
+
+test_under_flux 4
```

Comment:
Would an instance size=1 work for this test?

Reply:
Thanks! Just pushed the following changes
- `flux-module` calls exit(0) on SIGINT/SIGTERM
- module name is always printed
- test uses size=1

Reply:
Great! Feel free to set MWP when you're ready. 

Oh! Also maybe add bash completions, though I can add that as a follow-on PR because we probably forgot about `flux overlay trace` as well?

Reply:
Thanks!  Yeah I didn't think about that.  I'll go ahead and set MWP here as I'm headed out on an errand.